.buildkite/filebeat/scripts/integration-pytests.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/filebeat/scripts/common.sh
+
+# ToDo - remove after Beats agent is created"
+echo ":: Setup Env ::"
+add_bin_path
+with_go
+with_mage
+# ToDo - end
+
+echo ":: Execute Integration Tests ::"
+sudo chmod -R go-w filebeat/
+
+cd filebeat
+umask 0022
+mage goIntegTest
```

Comment:
I'm a bit confused here, the file is called `integration-pytests.sh` but it is running Go integration tests. I believe it should call the target for Python integration tests:
```suggestion
mage pythonIntegTest
```

Reply:
fixed